File: book-network/src/main/java/com/rpsh/booknetwork/book/BookResponse.java
package com.rpsh.booknetwork.book;


import com.rpsh.booknetwork.user.User;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class BookResponse {

    private Integer id;
    private String title;
    private String authorName;
    private String isbn;
    private String synopsis;
    private User owner;
    private byte[] cover;
    private double rate;
    private boolean archived;
    private boolean shareable;


}

File: book-network-ui/src/app/services/models/book-response.ts
/* tslint:disable */
/* eslint-disable */
import { User } from './user';
export interface BookResponse {
  archived?: boolean;
  authorName?: string;
  cover?: Array<string>;
  id?: number;
  isbn?: string;
  owner?: User;
  rate?: number;
  shareable?: boolean;
  synopsis?: string;
  title?: string;
}
